packages/forms/src/Form.js

```diff
@@ -88,7 +88,10 @@ class Form extends React.Component {
 	}
 
 	handleActionClick(onClick) {
```

Comment:
ok the transpilled version of this change proposal is not good at all ... let's keep this implemmentaiton